core/.js/src/main/scala/io/udash/bindings/Bindings.scala

```diff
@@ -411,21 +426,19 @@ object Bindings extends Bindings {
       * If callback returns true, other listeners which are queued will not be invoked.
       * If callback returns false, next callback in the queue will be invoked.
       */
-    def :+=[T <: Event](callback: (T) => Boolean): Modifier[Element] = {
+    def :+=[T <: Event](callback: T => Boolean): Modifier[Element] = {
       AttrPair(attr, callback, new AttrValue[Element, (T) => Boolean] {
```

Comment:
`(T)` has unnecessary braces (in general, `AttrValue` might be implemented with single method)

Reply:
It can't, this cross-compiles to 2.11